lib/strategy.js

```diff
@@ -171,7 +171,9 @@ AcpStrategy.prototype.authenticate = function(req, options = {}) {
 				null,
 				function (err, body, res) {
 	              if (err) {
-					  return self.error(new InternalOAuthError('failed to fetch user profile', err));
+					  console.log('Request to ', userInfoURL, ', failed ', res.statusCode);
```

Comment:
please safe get from `res` here. or check in oauth2 docs if `res` should always be present

Reply:
```suggestion
					  console.log('Request to ', userInfoURL, ', failed with status ', res.statusCode);
```